docker-compose.dev.yml

```diff
@@ -1,8 +1,15 @@
+#
+# docker-compose file used ONLY for local development.
```

Comment:
We could link to https://posthog.com/handbook/engineering/developing-locally#1-spin-up-external-services here for some more guidance